DOC-2105: Bring `changelog.md` into sync with `changelog.adoc` (#8891)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

## Unreleased

## 6.6.0 - 2023-07-12

### Added
- Added a new property value — `bottom` — for inline dialog configurations that anchors the dialog to the bottom of the editor. #TINY-9888
- Added a new property — `persistent` — for inline dialog configurations that will stop the dialog closing when clicking away from it. #TINY-9991
- New `ai`, `ai-prompt` and `send` icons. #TINY-9942
- Added a new property — `streamContent` — for the `iframe` dialog component. This causes `setData()` to update content without reloading the frame, and end scroll positions will be maintained as new content streams in. #TINY-10032
- AI Assistant plugin toolbar items added to the default toolbar and AI Assistant plugin menu items added to the default menu bar. #TINY-9939
- Added a new property — `border` — for the `iframe` dialog component that allows a border to be added. #TINY-10049
- Added a new property — `align` — for the label dialog component that controls text alignment. #TINY-10058

### Improved
- Displaying the footer section of a modal or inline dialog is now optional. #TINY-9996

### Improved

> - When defining a modal or inline dialog, if the `buttons` property is `undefined` or an empty array, the footer will now no longer be rendered. #TINY-9996

Author: ```suggestion
- When defining a modal or inline dialog, if the buttons property is an empty array, or is not defined at all, the footer will now no longer be rendered. #TINY-9996
```

## 6.6.0 - TBA

### Added

> - Added a new property — `bottom` — for inline dialog configurations. #TINY-9888

Reviewer: ```suggestion
- Added a new property — `bottom` — for inline dialog configurations that anchors the dialog to the bottom of the editor. #TINY-9888
```

> - Added a new property — `bottom` — for inline dialog configurations that anchors the dialog to the bottom of the editor. #TINY-9888

Author: ```suggestion
- Added a new property value — `bottom` — for inline dialog configurations that anchors the dialog to the bottom of the editor. #TINY-9888
```

> - Added a new property — `persistent` — for inline dialog configurations that controls dialog persistence when not in focus. #TINY-9991

Reviewer: I have a higher standard of docs than "turning on setting x controls the feature x behaviour". It's like saying the sky is blue because it has a blue colour.

```suggestion
- Added a new property — `persistent` — for inline dialog configurations that will stop the dialog closing when clicking away from it. #TINY-9991
```

- New `ai`, `ai-prompt` and `send` icons. #TINY-9942

> - Added a new property — `streamContent` — for the `iframe` dialog component which controls dialog `setData()` behavior. #TINY-10032

Reviewer: That's critical information, sorry. Entries either need details about the change or a link to more information. I've tried to shorten it further.

```suggestion
- Added a new property — `streamContent` — for the `iframe` dialog component. This causes `setData()` to update content without reloading the frame, and end scroll positions will be maintained as new content streams in. #TINY-10032
```

Author: > That's critical information, sorry. 

No problem. I’ll take the second sentence and put it into the release notes body copy for this entry.

- AI plugin toolbar items added to the default toolbar and AI plugin menu items added to the default menu bar. #TINY-9939

> - Added a new property — `border` — for the `iframe` dialog component that controls border visibility. #TINY-10049

Reviewer: ```suggestion
- Added a new property — `border` — for the `iframe` dialog component that allows a border to be added. #TINY-10049
```

> - AI plugin toolbar items added to the default toolbar and AI plugin menu items added to the default menu bar. #TINY-9939

Author: ```suggestion
AI Assistant plugin toolbar items added to the default toolbar and AI Assistant plugin menu items added to the default menu bar. #TINY-9939
```

Changed presentation name of new plugin, from _AI_ to _AI Assistant_ as per previously noted name-change.

> AI Assistant plugin toolbar items added to the default toolbar and AI Assistant plugin menu items added to the default menu bar. #TINY-9939

Author: ```suggestion
- AI Assistant plugin toolbar items added to the default toolbar and AI Assistant plugin menu items added to the default menu bar. #TINY-9939
```
Missing markup no longer missing.

- Added a new property — `align` — for the label dialog component that controls text alignment. #TINY-10058

### Improved
- Displaying the footer section of a modal or inline dialog is now optional. #TINY-9996

### Changed

> - The icon in an `alertbanner` dialog component is not clickable if the _URL_ field is not specified. #TINY-10013

Reviewer: ```suggestion
- The icon in an `alertbanner` dialog component is no longer clickable if the _URL_ field is not specified. #TINY-10013
```

### Fixed

> - Fixed an issue that prevented setting the inline dialog `size` to _medium_ or _normal_. #TINY-10015

Reviewer: ```suggestion
- Fixed an issue that caused the inline dialog `size` setting to have no effect. #TINY-10015
```
